README.md

```diff
@@ -50,7 +50,7 @@ This is a list of volumes that belong to the current pool. It follows the
 same pattern as the `storage_volumes` variable, explained below.
 
 ##### `encryption`
-This specifies whether or not the pool will be encrypted using LUKS.
+This specifies whether the pool will be encrypted using LUKS.
```

Comment:
I would prefer "This specifies that the pool . . ." - I prefer indicative rather than conditional or subjunctive